util/qubes/qubes.lisp

```diff
@@ -0,0 +1,81 @@
+;;;; qubes.lisp
+
+(in-package #:qubes)
+
+;; Qubes module for StumpWM.
+;;
+;; Copyright (C) 2016 Johanna Abrahamsson
+;;
+;; Maintainer: Johanna Abrahamsson
+;;
+
+;;; Code:
+
+(defun alloc-color-on-current (color)
+  (stumpwm::alloc-color (current-screen) color))
+
+
+(defparameter *label-colors* 
+      (list (alloc-color-on-current "pink1")
+            (alloc-color-on-current "red1")
+            (alloc-color-on-current "orange1")
+            (alloc-color-on-current "yellow1")
+            (alloc-color-on-current "green1")
+            (alloc-color-on-current "gray1")
+            (alloc-color-on-current "blue1")
+            (alloc-color-on-current "purple1")
+            (alloc-color-on-current "black")))
+
+
+(defun window-label-color (window)
+  (nth (or 
+         (first (xlib:get-property (window-xwin window) :_QUBES_LABEL)) 
+         0) *label-colors*))
```

Comment:
The `(first ...` could go right after the `(or ` and the `*label-colors*` should go on its own line. Like this:

```lisp
(defun window-label-color (window)
  (nth (or (first (xlib:get-property (window-xwin window) :_QUBES_LABEL))
           0)
       *label-colors*))
```